xwiki-platform-distribution/xwiki-platform-distribution-debian/xwiki-platform-distribution-debian-xjetty/xwiki-platform-distribution-debian-xjetty-mysql/src/deb/control/config

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+set -e
+#set -x
```

Comment:
Should be removed no?

Reply:
It's in all our packages sources, used when debuging stuff.